howto/install-appliance/enable-oidc.md

```diff
@@ -0,0 +1,55 @@
+(howto-appliance-enable-oidc)=
```

Comment:
Should this be under `howto/install-appliance`? If this is a mandatory step of the installation, I would say yes. However, it seems like an optional provision for users who would like to enable OIDC. Wouldn't it be better to have this under `howto/anbox` similar to setting up `tls-for-appliance.md`?

Reply:
It is not mandatory, more an option to the installation. We can move to `howto/anbox` but it feels more like something we want to highlight as an installation option.

Reply:
Also `tls-for-appliance.md` is currently written more as a post-install topic.

Reply:
Okay, we can keep it here now. I will take another look at the TLS topic and see if we can group topics related to installation together. I do see a concern that keeping things under "Manage Anbox Cloud" could be a bit unintuitive for things that can be done during installation.